locales/en/translation.json

```diff
@@ -1,34 +1,961 @@
+  "Don't lose it!": "Don't lose it!",
+  "In order to protect your funds from being accessible to hackers and thieves, the recovery phrase must be kept somewhere safe and remain a secret.": "In order to protect your funds from being accessible to hackers and thieves, the recovery phrase must be kept somewhere safe and remain a secret.",
+  "I Understand": "I Understand",
+  "Next": "Next",
+  "Encrypt your Key": "Encrypt your Key",
+  "Prevent an unauthorized user from sending funds out of your wallet.": "Prevent an unauthorized user from sending funds out of your wallet.",
+  "Key Settings": "Key Settings",
+  "View all the ways to manage and configure your key.": "View all the ways to manage and configure your key.",
+  "Key Options": "Key Options",
+  "Other Keys": "Other Keys",
+  "To continue please enter your encryption password.": "To continue please enter your encryption password.",
+  "New wallet found": "New wallet found",
+  " wallets found": " wallets found",
+  "Your key is already synced": "Your key is already synced",
+  "Sync wallet": "Sync wallet",
+  "Failed to Sync wallets": "Failed to Sync wallets",
+  "Key Name": "Key Name",
+  "Wallets": "Wallets",
+  "Backup": "Backup",
+  "Request Encrypt Password": "Request Encrypt Password",
+  "Password Not Recoverable": "Password Not Recoverable",
+  "This password cannot be recovered. If this password is lost, funds can only be recovered by reimporting your 12-word recovery phrase.": "This password cannot be recovered. If this password is lost, funds can only be recovered by reimporting your 12-word recovery phrase.",
+  "Clear Encrypt Password": "Clear Encrypt Password",
+  "Advanced": "Advanced",
+  "Sync Wallets Across Devices": "Sync Wallets Across Devices",
+  "Create a Shared Wallet": "Create a Shared Wallet",
+  "Use more than one device to create a multisig wallet": "Use more than one device to create a multisig wallet",
+  "Join a Shared Wallet": "Join a Shared Wallet",
+  "Joining another user's multisig wallet requires an invitation to join": "Joining another user's multisig wallet requires an invitation to join",
+  "Buy ": "Buy ",
+  "Don't risk losing your money": "Don't risk losing your money",
+  "Your recovery key is composed of 12 randomly selected words. Take a couple of minutes to carefully write down each word in the order they appear.": "Your recovery key is composed of 12 randomly selected words. Take a couple of minutes to carefully write down each word in the order they appear.",
+  "I'M SURE": "I'M SURE",
+  "Write down each word.": "Write down each word.",
+  "Payment Request": "Payment Request",
+  "Share this QR code to receive in your wallet .": "Share this QR code to receive in your wallet .",
```

Comment:
`Share this QR code to receive {{amountUnitStr}} in your wallet {{walletName}}.`